python_modules/libraries/dagster-dg/dagster_dg/cli/vscode_utils.py

```diff
@@ -0,0 +1,89 @@
+import json
```

Comment:
Can we put this file in `dagster_dg/utils/vscode/__init__.py` instead? `cli` module is just the CLI layer.